spec/buffered-process-spec.coffee

```diff
@@ -116,29 +112,25 @@ describe "BufferedProcess", ->
       expect(stderr).toContain 'apm - Atom Package Manager'
       expect(stdout).toEqual ''
 
-  it "calls the specified stdout callback only with whole lines", ->
+  it "calls the specified stdout callback with whole lines", ->
     exitCallback = jasmine.createSpy('exit callback')
-    baseContent = "There are dozens of us! Dozens! It's as Ann as the nose on Plain's face. Can you believe that the only reason the club is going under is because it's in a terrifying neighborhood? She calls it a Mayonegg. Waiting for the Emmys. BTW did you know won 6 Emmys and was still canceled early by Fox? COME ON. I'll buy you a hundred George Michaels that you can teach to drive! Never once touched my per diem. I'd go to Craft Service, get some raw veggies, bacon, Cup-A-Soup…baby, I got a stew goin'"
-    content = (baseContent for _ in [1..200]).join('\n')
+    loremPath = require.resolve("./fixtures/lorem.txt")
+    content = fs.readFileSync(loremPath).toString()
+    baseContent = content.split('\n')
     stdout = ''
-    endLength = 10
-    outputAlwaysEndsWithStew = true
+    allLinesEndWithNewline = true
     process = new BufferedProcess
-      command: '/bin/echo'
-      args: [content]
+      command: if platform.isWindows() then 'type' else 'cat'
```

Comment:
We're going to support x64 at some point...

On Mon, May 2, 2016 at 7:48 PM Wliu <notifications@github.com> wrote:

> In spec/buffered-process-spec.coffee
> <https://github.com/atom/atom/pull/11642#discussion_r61833555>:
>
> >      process = new BufferedProcess
> > -      command: '/bin/echo'
> > -      args: [content]
> > +      command: if platform.isWindows() then 'type' else 'cat'
>
> process.platform is 'win32'? Then the extra platform require can be
> removed.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/atom/atom/pull/11642/files/7542a401fff99d6ca93e7b1254bd8fc4d3672559#r61833555>
>